constants/routes.constant.js

```diff
@@ -150,6 +150,41 @@ const hackerRoutes = {
     }
 };
 
+const travelRoutes = {
+    getSelf: {
+        requestType: Constants.REQUEST_TYPES.GET,
+        uri: "/api/travel/self/"
+    },
+    getSelfById: {
+        requestType: Constants.REQUEST_TYPES.GET,
+        uri: "/api/travel/" + Constants.ROLE_CATEGORIES.SELF
+    },
+    getAnyById: {
+        requestType: Constants.REQUEST_TYPES.GET,
+        uri: "/api/travel/" + Constants.ROLE_CATEGORIES.ALL
+    },
+    getSelfByEmail: {
+        requestType: Constants.REQUEST_TYPES.GET,
+        uri: "/api/travel/email/" + Constants.ROLE_CATEGORIES.SELF
+    },
+    getAnyByEmail: {
+        requestType: Constants.REQUEST_TYPES.GET,
+        uri: "/api/travel/email/" + Constants.ROLE_CATEGORIES.ALL
+    },
+    post: {
+        requestType: Constants.REQUEST_TYPES.POST,
+        uri: "/api/travel/"
+    },
```

Comment:
what is this post? Did you mean create travel?

Reply:
Yeah, it's for create - thought I was following the naming convention from account and hacker, would it be better to rename this from `post` to `createTravel`?